src/bundle/makeRepairDataProperties.js

```diff
@@ -4,19 +4,17 @@
 // https://github.com/google/caja/blob/master/src/com/google/caja/ses/repairES5.js
 
 export default function makeRepairDataProperties() {
+  // Object.defineProperty is allowed to fail silently,
+  // use Object.defineProperties instead.
+
   const {
     defineProperties,
+    getOwnPropertyDescriptor,
     getOwnPropertyDescriptors,
     hasOwnProperty,
```

Comment:
How can this even seem to work? `Object` has no `.hasOwnProperty`. That's only on `Object.prototype`. OMG, it happens to work because `Object` inherits it from `Object.prototype`. Although this happens to work, it must be fixed.

Reply:
TIL :eyes: 
```js
Object.getPrototypeOf(Object.getPrototypeOf(Object)) === Object.prototype
```

Reply:
Fixed.